bin/collator/src/local/chain_spec.rs

```diff
@@ -194,6 +194,7 @@ pub(crate) mod tests {
     use sp_runtime::BuildStorage;
 
     #[test]
+    #[ignore]
```

Comment:
Why this btw? 🙂

Reply:
ok I will let it compile local runtime, will revert this